consent/sql_helper.go

```diff
@@ -408,7 +408,7 @@ func (s *sqlHandledAuthenticationRequest) toHandledAuthenticationRequest(a *Auth
 	return &HandledAuthenticationRequest{
 		ForceSubjectIdentifier: s.ForceSubjectIdentifier,
 		RememberFor:            s.RememberFor,
-		Remember:               s.Remember,
+		Remember:               false,
```

Comment:
That does not seem right. Why would this always be false? And why only for SQL? This should be done in the handler instead for requests for `Skip` is false. This change needs to be reverted.